sklearn/compose/tests/test_column_transformer.py

```diff
@@ -2248,6 +2275,32 @@ def test_remainder_set_output():
     assert isinstance(out, np.ndarray)
 
 
+def test_dataframe_different_dataframe_libraries():
```

Comment:
do we wanna test for when train is done with polars but transform with pandas / other way around?